new_ui/templates/partials/settings-drawer.html

```
@@ -0,0 +1,251 @@
+                @click="$store.ui.toggleSettings()"
+                aria-label="Close settings">
+            ✕
+        </button>
+    </div>
+
+    <!-- Feature Toggles -->
+    <div class="settings-drawer__toggles">
+        <div class="settings-drawer__toggle-row">
+            <span class="text-heading-base">Input Image</span>
+            <label class="toggle">
+                <input type="checkbox" x-model="inputImageEnabled">
```

HTMLHint (1.9.2)

[warning] No matching [ label ] tag found.

(input-requires-label)